### DISCUSSION TITLE
Manual target build issue

### DISCUSSION THREAD

**Discussion by UT3ULJ** [iNavFlight/inav#9189] [Q&A]
Hi, i'm trying to figure out how to build firmware by myself. I was followed the instruction in the docs and get working VSCode under WSL on Win10.
But can't make a build for f7 targets...it says there flash overflow.. Here is output with an error:
![зображення](https://github.com/iNavFlight/inav/assets/44117507/aa18b4ba-0917-4d6a-b19c-e106d44e4c5e)

And in the same time MATEKF405 target is build succesfully... What i'm doing wrong?
![зображення](https://github.com/iNavFlight/inav/assets/44117507/0e0ae27d-a06a-48b5-bc11-c1d391c65d09)

**Comment by MrD-RC** [comment]
Have you made any code changes? The flash file is too big to fit on the F722. It’s using 119% of the available flash space.

**Reply by UT3ULJ** [reply]
Thanks for reply! No, i just download the repo from git and just trying to build the target...  Maybe i should try to redownload it... but it's strange..

**Reply by UT3ULJ** [reply]
I found that if i just download the git repo and then try to build any target, everything works fine.

![зображення](https://github.com/iNavFlight/inav/assets/44117507/9cfd8d13-b166-4956-9304-af83d6157ca3)

but when i folow the steps from this instruction (https://github.com/iNavFlight/inav/blob/master/docs/development/IDE%20-%20Visual%20Studio%20Code%20with%20Windows%2010.md) and trying build from VSCode, i'm start getting the error from the topic.. Maybe it's a build optimization level problem or something... I don't know where i can set it
